### PR TITLE
Picard 2.0.3

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

File: org.musicbrainz.Picard.appdata.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
   </content_rating>
 
   <releases>
+    <release date="2018-08-10" version="2.0.3"/>
```
Comment (bochecha) [inline org.musicbrainz.Picard.appdata.xml]: Eventually, we should try and have some release notes here.

Maybe for future versions. :slightly_smiling_face:

**Comment by bochecha** [comment]
bot, build

**Comment by bochecha** [comment]
Since you're staying on top of releases much better than me these days, want to have push access to this repository?

We can continue working with pull requests anyway as we are, but at least:

1. you can put your branches directly in this repo, no need for you to maintain a fork
2. you can ask buildbot to make test builds (the `bot, build` thing I used above)

**Comment by Lctrs** [comment]
Sure !

**Comment by bochecha** [comment]
@Lctrs you should have push access now. Try merging your pull request!

**Comment by Lctrs** [comment]
Done. Thanks :+1:

**Pull request merged**